packages/types/src/events/reaction.ts

```diff
@@ -0,0 +1,23 @@
+interface ReactionMessageItem {
```

Comment:
Should this be exported? Also, since this is a `message` type, should it be in message? That message file was the one that gave me the most pause. 😅

Reply:
It should not be exported, as it is only used in this file, which models reaction-event payloads.

Reply:
Got it! Skimmed these so missed the usage, but that makes sense.